spec/features/post_auth_spec.rb

```diff
@@ -129,21 +122,26 @@
       end
     end
 
+    context 'Access-Accept' do
+      let(:authentication_result) { 'Access-Accept' }
+
+      it_behaves_like 'it saves the right logging information'
+
+      it 'updates the user last login' do
+        post_auth_request
+        expect(user.last_login).to_not be_nil
+      end
+    end
+
     context 'Access-Reject' do
       let(:authentication_result) { 'Access-Reject' }
 
-      it 'does not record a session' do
-        expect(Session.count).to eq(0)
-      end
+      it_behaves_like 'it saves the right logging information'
 
-      it 'does not record last_login for the user' do
+      it 'updates the user last login' do
```

Comment:
You've updated the test name here, but not the test implementation.  Looks like a mistake.